phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -1855,7 +1855,7 @@ public function format_property_text($value, $tag) {
         $tempLinkValue = str_replace(
```

Comment:
The replace list here differs from other point. Maybe there is necessary two normalize functions, or if the difference is accidental, to test what list of substitutions is the correct.